src/Vinesauce ROM Corruptor/Components/ByteCorruptionTextBox.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Windows.Forms;
+
+namespace Vinesauce_ROM_Corruptor.Components
+{
+    /// <summary>
+    /// Custom TextBox for the byte corruption settings
+    /// </summary>
+    public sealed class ByteCorruptionTextBox : TextBox
```

Comment:
This class works as intended if pressing keys was the only way to insert text. Unfortunately we load the text directly into the box when loading from a settings string (ie. file or URL). This fact, coupled with the removal of the try/catch statements makes it possible to crash the program using a settings string with bad values.